src/qibo_cloud_backends/braket_client.py

```diff
@@ -0,0 +1,64 @@
+import time
+
+from braket.aws import AwsDevice
+from braket.devices import LocalSimulator
+from qibo import Circuit as QiboCircuit
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes
+
+from qibo_cloud_backends.braket_translation import to_braket
+
+
+class BraketClientBackend(NumpyBackend):
+    def __init__(self, device=None, verbatim_circuit=False, monitoring=False):
```

Comment:
```suggestion
    def __init__(self, device=None, verbatim_circuit=False, verbosity=False):
```
Sorry I just noticed that we are using the name `verbosity` in the other backends, could you switch the name for the `braket-client` as well? In general it would be good to uniform as much as possible all the backends. Even `device` could be changed in to `platform`, maybe, but this one is not strictly required as it's not exactly the same thing...

Reply:
Will do!